File: Retos/Reto #20 - LA TRIFUERZA [Media]/python/albertovf.py
def trifuerza(nivel:int):
    longitud = 2*nivel

    n = 1
    while (n != nivel+1):
        linea = '*'*(2*n-1)
        linea=linea.center(longitud*2)
        print(linea)
        n+=1

    n = 1
    while (n != nivel+1):
        linea = '*' *(2*n-1)
        linea=linea.center(longitud) + linea.center(longitud)
        print(linea)
        n+=1

trifuerza(10)

File: Retos/Reto #20 - LA TRIFUERZA [Media]/typescript/albertovf.ts
function trifuerza(nivel: number): void {
	const longitud: number = 2 * nivel;

	for (let n = 1; n <= nivel; n++) {
		let linea: string = '*'.repeat(2 * n - 1);
		linea = linea.padStart((longitud * 2 - linea.length) / 2 + linea.length);
		console.log(linea);
	}

	for (let n = 1; n <= nivel; n++) {
		let linea: string = '*'.repeat(2 * n - 1);
		let bordes = ' '.repeat(nivel - n);
		linea = bordes + linea + bordes + bordes + ' ' + linea;
		console.log(linea);
	}
}

trifuerza(5)